tsconfig.prod.json

```diff
@@ -13,17 +15,18 @@
   },
   "files": [],
   "references": [
+    // Order matters
```

Comment:
@piotr-roslaniec Was this auto-generated? - JSON doesn't allow for comments.

Reply:
It's not autogenerated. Yes, JSON doesn't allow comments, but `tsconfig.json` is not a strict JSON and it [allows comments](https://github.com/microsoft/TypeScript/issues/4987).

Reply:
👍 - Interesting